### PR TITLE
SSC - Octane integration - inject SSC vulnerabilities to Octane -us #587042 : Fortify OnPrem E2E

### PR DESCRIPTION
**Pull request by yamin-hijazi**
SSC integration code.

### CODE REVIEW

**Comment by yamin-hijazi** [comment]
Reviewed by Radi.

**Comment by m-seldin** [comment]
@yamin-hijazi 
please check appveyor build failures
please check codacy errors
please enter proper pull request title and description (with jira and octane issues numbers)

File: src/main/java/com/microfocus/application/automation/tools/octane/configuration/SSCServerConfigUtil.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
+package com.microfocus.application.automation.tools.octane.configuration;
```
Comment (m-seldin) [inline src/main/java/com/microfocus/application/automation/tools/octane/configuration/SSCServerConfigUtil.java:22]: @yamin-hijazi 
add license

File: src/main/java/com/microfocus/application/automation/tools/octane/configuration/SSCServerConfigUtil.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+import hudson.model.Descriptor;
+import jenkins.model.Jenkins;
+
+import java.lang.reflect.Field;
+
+public class SSCServerConfigUtil {
```
Comment (m-seldin) [inline src/main/java/com/microfocus/application/automation/tools/octane/configuration/SSCServerConfigUtil.java:31]: add class documentation

**Review by m-seldin — CHANGES_REQUESTED** [review]
please correct changes in comments

**Comment by m-seldin** [comment]
appveyor still failing.
no jira issue

**Pull request merged**